tools/benchmark/cmd/mvcc-put.go

```diff
@@ -106,16 +108,25 @@ func mvccPutFunc(cmd *cobra.Command, args []string) {
 	rrc := r.Results()
 
 	rc := r.Run()
-	for i := 0; i < totalNrKeys; i++ {
-		st := time.Now()
-		if txn {
+
+	if txn {
+		for i := 0; i < totalNrKeys; i += nrBatch {
```

Comment:
instead of having `totalNrKeys`, have `mvccTotalRequests` like `txn-put`? avoids having to account for individual keys when only interested in iterating on requests.